x-pack/packages/ml/data_frame_analytics_utils/src/constants.ts

```diff
@@ -36,6 +36,7 @@ export const DEFAULT_RESULTS_FIELD = 'ml';
  */
 export const JOB_MAP_NODE_TYPES = {
   ANALYTICS: 'analytics',
+  ANALYTICS_JOB_MISSING: 'analycs-job-missing',
```

Comment:
Looks like a typo here `analycs`

Reply:
Updated in b65e50782fd9bd90e1ef49f5de3f9b1aaed495bd